webodf/lib/gui/SessionController.js

```diff
@@ -116,6 +116,13 @@ gui.SessionControllerOptions = function () {
             "Expected to be run in an environment which has a global window, like a browser.");
 
         /**
+         * @return {!gui.SessionControllerOptions}
+         */
+        this.getConfiguration = function() {
```

Comment:
This is not used anywhere and exposes the internals of SessionController and makes it possible to change them.
If the configuration should be exposed, please expose a read-only version or make a copy (would could get outdated).